assets/scss/_custom.scss

```diff
@@ -639,3 +639,10 @@ body.td-documentation {
   }
 }
 
+
+// Remove shadow from code in table which is added by Docsy
+.td-content {
+  p code, li > code, table code {
+    background-color: inherit !important;
```

Comment:
For accessibility this needs to set `color` as well:
```
    color: inherit !important;
    background-color: inherit !important;
```
We can fix that in a follow-up PR I think.